src/rspamd.c

```diff
@@ -781,9 +781,6 @@ static void
 mark_old_workers (gpointer key, gpointer value, gpointer unused)
 {
 	struct rspamd_worker *w = value;
-	struct rspamd_main *rspamd_main;
-
-	rspamd_main = w->srv;
 
```

Comment:
`rspamd_main` is initialised for logging here like `msg_*_main`. So I'd not recommend to remove this code.

Reply:
Ok, added an attribute unused and added fix from mumhash upstream as well